<?xml version="1.0"?>
<launch>
  <arg name="speed_scale" default="1.0"/>
  <node pkg="denso_gazebo_grasp" type="move_and_grasp" name="move_and_grasp" output="screen">
    <param name="speed_scale" value="$(arg speed_scale)"/>
  </node>
</launch>